### ISSUE TITLE
Visualize and explore panels v0.2.40

### ISSUE THREAD

**Issue by kmezhoud** [vnijs/radiant#22]
HI Vincent,
I installed the last version V0.2.40.
I note in Visualize sidebar, the dropdown variables are missing (also explore panel).
The new checkbox(Line, Loess, Jitter) are visible only once. 
Maybe these news features are in progress..
Thanks
Karim

**Comment by vnijs** [comment]
Thanks for reporting @kmezhoud Did you you pull from github, install using devtools, or did you use install.packages("radiant", repos = "http://vnijs.github.io/radiant_miniCRAN/")?

**Comment by vnijs** [comment]
@kmezhoud Could you try the commit I just pushed to github?

```r
devtools::install_github("vnijs/radiant")
library(radiant)
radiant()
```

**Comment by kmezhoud** [comment]
Hi,
Not changes: https://drive.google.com/file/d/0B9NOY9eukkEeRU5iMzNTQWxiVEk/view?usp=sharing
I thing the issue comes from this modification:

 # vars <- varnames()
  vars <- varying_vars()
Thanks

**Comment by vnijs** [comment]
Can you send me the output of:

`sessionInfo()`

Thanks

**Comment by kmezhoud** [comment]
Restarting R session...

> library(radiant)
Loading required package: magrittr
Loading required package: ggplot2
Loading required package: tidyr

Attaching package: ‘tidyr’

The following object is masked from ‘package:magrittr’:

    extract

Loading required package: dplyr

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union


Attaching package: ‘radiant’

The following object is masked from ‘package:ggplot2’:

    diamonds

> sessionInfo()
R version 3.2.1 (2015-06-18)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.4 (Yosemite)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets 
[6] methods   base     

other attached packages:
[1] radiant_0.2.41 dplyr_0.4.2    tidyr_0.2.0   
[4] ggplot2_1.0.1  magrittr_1.5  

loaded via a namespace (and not attached):
 [1] wordcloud_2.5         slam_0.1-32          
 [3] reshape2_1.4.1        splines_3.2.1        
 [5] lattice_0.20-31       colorspace_1.2-6     
 [7] htmltools_0.2.6       yaml_2.1.13          
 [9] mgcv_1.8-6            AlgDesign_1.1-7.3    
[11] nloptr_1.0.4          DBI_0.3.1            
[13] RColorBrewer_1.1-2    pryr_0.1.2           
[15] plyr_1.8.3            GPArotation_2014.11-1
[17] stringr_1.0.0         munsell_0.4.2        
[19] gtable_0.1.2          htmlwidgets_0.5      
[21] codetools_0.2-11      psych_1.5.6          
[23] memoise_0.2.1         knitr_1.10.5         
[25] SparseM_1.6           httpuv_1.3.2         
[27] quantreg_5.11         pbkrtest_0.4-2       
[29] parallel_3.2.1        markdown_0.7.7       
[31] broom_0.3.7           proto_0.3-10         
[33] Rcpp_0.11.6           xtable_1.7-4         
[35] scales_0.2.5          DT_0.1               
[37] mime_0.3              lme4_1.1-8           
[39] gridExtra_0.9.1       mnormt_1.5-3         
[41] rpivotTable_0.1.4     digest_0.6.8         
[43] stringi_0.5-5         shiny_0.12.1         
[45] grid_3.2.1            tools_3.2.1          
[47] MathJaxR_0.11         ggdendro_0.1-15      
[49] car_2.0-25            shinyAce_0.2.1       
[51] MASS_7.3-42           Matrix_1.2-1         
[53] lubridate_1.3.3       assertthat_0.1       
[55] minqa_1.2.4           R6_2.1.0             
[57] nnet_7.3-10           nlme_3.1-121

**Comment by vnijs** [comment]
I cannot replicate the issue on mac but I do see it on Windows. On windows the issue only pops-up when I do not have launch.browser = TRUE set. Can you try the following to launch radiant?

`shiny::runApp('inst/base', launch.browser = TRUE)`

and 

`library(radiant); radiant()`

**Comment by kmezhoud** [comment]
OK,
That works fine with Firefox.
Thanks

**Comment by vnijs** [comment]
It also works fine in Chrome and Safari. I wonder if this might be related to an issue I filled at rstudio/shiny yesterday. Would you mind trying out the mini-shiny app mentioned in the issue linked below? I know it says the issue happens on Windows only but perhaps you could replicate on you mac. Thanks!

https://github.com/rstudio/shiny/issues/894

**Comment by vnijs** [comment]
I update Radiant to use the new version of gridExtra. The issue with Visualize and Explore remains but on when using Rstudio's viewer. All works fine in a browser. See also the version deployed on shiny-apps. Looks fine. I will close this for now as it seems this is might be an Rstudio viewer issue. Thanks for reporting and testing.

https://vnijs.shinyapps.io/base

**Comment by vnijs** [comment]
@kmezhoud Can you check if you still have this issue with the current release? Thanks.

**Comment by kmezhoud** [comment]
Hi,
 i am testing radiant 0.2.51. this is just for your knowledge.


  devtools::install_github("vnijs/radiant")
Downloading github repo vnijs/radiant@master
Installing radiant
Skipping 1 packages not available: MathJaxR
Skipping 1 packages ahead of CRAN: shinyAce
'/Library/Frameworks/R.framework/Resources/bin/R' --no-site-file  \
  --no-environ --no-save --no-restore CMD INSTALL  \
  '/private/var/folders/cd/kys1ttrn0mzggqxzzhpl1mw00000gp/T/RtmpZjAtc3/devtools19f3da71925/vnijs-radiant-00b9e29'  \
  --library='/Library/Frameworks/R.framework/Versions/3.2/Resources/library'  \
  --install-tests 

* installing *source* package ‘radiant’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** tests
** preparing package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  namespace ‘DT’ 0.1 is being loaded, but >= 0.1.26 is required
ERROR: lazy loading failed for package ‘radiant’
* removing ‘/Library/Frameworks/R.framework/Versions/3.2/Resources/library/radiant’
* restoring previous ‘/Library/Frameworks/R.framework/Versions/3.2/Resources/library/radiant’
Error: Command failed (1)

**Comment by vnijs** [comment]
Installing from github would not install dependencies on github. You can install the latest version of DT from github first using
devtools::install_github("rstudio/DT")

or just install radiant using: 
install.packages("radiant", repos = "http://vnijs.github.io/radiant_miniCRAN/")

**Comment by kmezhoud** [comment]
@vnijs This issue is solved  with radiant 0.2.51.
> sessionInfo()
R version 3.2.1 (2015-06-18)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.4 (Yosemite)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shiny_0.12.1   radiant_0.2.51 dplyr_0.4.2    tidyr_0.2.0    ggplot2_1.0.1  magrittr_1.5  

loaded via a namespace (and not attached):
 [1] Rcpp_0.11.6           lubridate_1.3.3       lattice_0.20-31       assertthat_0.1        digest_0.6.8          psych_1.5.6           mime_0.3              slam_0.1-32          
 [9] R6_2.1.0              plyr_1.8.3            AlgDesign_1.1-7.3     lazyeval_0.1.10       rstudioapi_0.3.1      minqa_1.2.4           SparseM_1.6           car_2.0-25           
[17] nloptr_1.0.4          R.oo_1.19.0           Matrix_1.2-1          DT_0.1.28             rmarkdown_0.7         proto_0.3-10          splines_3.2.1         lme4_1.1-8           
[25] readr_0.1.1           stringr_1.0.0         htmlwidgets_0.5       munsell_0.4.2         broom_0.3.7           httpuv_1.3.2          GPArotation_2014.11-1 mnormt_1.5-3         
[33] shinyAce_0.2.1        mgcv_1.8-6            htmltools_0.2.6       nnet_7.3-10           gridExtra_2.0.0       codetools_0.2-11      MASS_7.3-42           R.methodsS3_1.7.0    
[41] grid_3.2.1            nlme_3.1-121          jsonlite_0.9.16       xtable_1.7-4          gtable_0.1.2          DBI_0.3.1             scales_0.2.5          stringi_0.5-5        
[49] rpivotTable_0.1.4     MathJaxR_0.11         pryr_0.1.2            reshape2_1.4.1        ggdendro_0.1-15       wordcloud_2.5         RColorBrewer_1.1-2    tools_3.2.1          
[57] markdown_0.7.7        yaml_2.1.13           parallel_3.2.1        pbkrtest_0.4-2        colorspace_1.2-6      memoise_0.2.1         knitr_1.10.5          quantreg_5.11

**Comment by vnijs** [comment]
Excellent. Thanks for confirming @kmezhoud
